shell/imports/server/migrations.js

```diff
@@ -787,6 +787,49 @@ function setIpBlacklist(db, backend) {
   }
 }
 
+function notifyIdentityChanges(db, backend) {
+  // Notify users who might be affected by the identity model changes.
+  //
+  // Two types of users are affected:
+  // - Users who have multiple identities with differing names.
+  // - Users who have identities that are shared with other users.
+  //
+  // However, the second group seems like it must be a subset of the first group. So we only check
+  // for the first.
+
+  const names = {};
+  Meteor.users.find({ "profile.name": { $exists: true } }, { fields: { "profile.name": 1 } })
+      .forEach(user => {
```

Comment:
I think in practice almost all identities will have the profile filled in since you're forced to submit your profile immediately after first login.

I guess adding calls to `SandstormDb.fillInProfileDefaults()` would not be very hard but, again, not sure it's worth spending time to re-test, so, meh. Maybe some extra people will get notifications.